Added systemd script example (#7)
anycable/docs.anycable.io: docs/go_getting_started.md

=> INFO time context=http Starting HTTPS server at 0.0.0.0:443
```

## Deploying
### Docker

Official docker images available at [DockerHub](https://hub.docker.com/r/anycable/anycable-go/).

### Heroku

See [heroku-anycable-go](https://github.com/anycable/heroku-anycable-go) buildpack.

### Systemd

> To run on systemd you can:

Reviewer: I think it's better to expand a bit and describe _why_ we recommend to run anycable-go as a service.

Something like this:
> We recommend running anycable-go as a system service on production servers for better manageability. On most modern Linux distributions this can be done by declaring a systemd service like this:

Author: Added

1. Edit as needed and save the following script to `/etc/systemd/system/anycable-go.service`

> 2. Reload services via `sudo systemctl daemon-reload`

Reviewer: Strictly speaking, this command makes systemd to re-read its configuration and notice a new file:
```suggestion
2. Reload systemd configuration via `sudo systemctl daemon-reload`
```

> 3. Run as usual - `sudo systemctl start anycable-go`

Reviewer: ```suggestion
3. Start anycable-go as a service: `sudo systemctl start anycable-go`
```

> ```

Reviewer: Let's add syntax highlighting!
```suggestion
```ini
```
Not sure whether it will affect resulting documentation, but at least it will help at GitHub.

As per [docs](https://www.freedesktop.org/software/systemd/man/systemd.unit.html):
> A unit file is a plain text ini-style file

Author: It will not, but agree
